app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -70,6 +70,8 @@ function initReactions() {
 
       for (var i = 0; i < butts.length; i++) {
         var butt = butts[i];
+        butt.setAttribute('aria-pressed', butt.classList.contains('reacted'));
```

Comment:
This attribute is used to communicate to assistive tech whether or not the button is currently selected (i.e. whether the user has "liked" the comment or not)

Reply:
The return of the butts! 🤣

Reply:
I know right 🤣  So delighted to be able to dip into the butts code one last time this year

Reply:
Reviewing this PR is bringing me absolute joy!  The 3rd grader in me is all giggles.